lib/Command/Configure.php

```diff
@@ -217,6 +217,26 @@ private function configureSms(InputInterface $input, OutputInterface $output) {
 				$providerConfig->setDid($did);
 				break;
 
+			case 'voipbuster':
+				$config->setProvider($provider);
+		
+				/** @var VoipMsConfig $providerConfig */
```

Comment:
import of this class is missing

Reply:
> import of this class is missing

My very first pull request; and a complete noob in GitHub ;) 
What do you mean? I forgot to sync the ProviderFactory.php aswell from my nextcloud instance, which handles the setProvider statement.

Reply:
No problem :)

There is no `use ...` statement at the head of the file for this class. See how the other providers import their config.

Please push the provider factory change as well

Reply:
Added the class definition in the head of the file.